scoring.js

```diff
@@ -0,0 +1,48 @@
+function calculatePassingScore(player) {
+  const touchdowns = player.stats.passing.touchdowns * 6
+  const interceptions = player.stats.passing.interceptions * -3
+  const yards = player.stats.passing.yards / 25
+
+  return touchdowns + interceptions + yards
+}
+
+function calculateRushingScore(player) {
+  const yards = player.stats.rushing.yards / 10
+  const touchdowns = player.stats.rushing.touchdowns * 6
+  const fumbles = player.stats.rushing.fumbles * -3
+  return yards + touchdowns + fumbles
+}
```

Comment:
Try to watch the consistency with your spacing. I like the extra line in calculatePassingScore, helps make it readable, but your didn't then do that in calculateRushingScore. With code, consistency is key.